internal/pkg/v2/infrastructure/redis/device.go

```diff
@@ -19,15 +19,15 @@ import (
 )
 
 const (
-	DeviceCollection            = "v2:device"
-	DeviceCollectionName        = DeviceCollection + ":" + v2.Name
-	DeviceCollectionLabel       = DeviceCollection + ":" + v2.Label
-	DeviceCollectionServiceName = DeviceCollection + ":" + v2.Service + ":" + v2.Name
+	DeviceCollection            = "md|dev"
```

Comment:
could we consider to use `dv` instead of `dev` to represent device?

Reply:
Modified.